pkg/revocation/revocation_db.go

```diff
@@ -49,38 +71,38 @@ func NewRevocationDB(revocationDBURL string) (*RevocationDB, error) {
 	return db, nil
 }
 
-// newRevocationDBBolt creates a bolt-backed RevocationDB
-func newRevocationDBBolt(path string) (*RevocationDB, error) {
+// newRevocationDBBolt creates a bolt-backed DB
+func newRevocationDBBolt(path string) (*DB, error) {
 	client, err := boltdb.New(path, extensions.RevocationBucket)
 	if err != nil {
 		return nil, err
 	}
-	return &RevocationDB{
-		DB: client,
+	return &DB{
+		KVStore: client,
 	}, nil
 }
 
-// newRevocationDBRedis creates a redis-backed RevocationDB.
-func newRevocationDBRedis(address string) (*RevocationDB, error) {
+// newRevocationDBRedis creates a redis-backed DB.
+func newRevocationDBRedis(address string) (*DB, error) {
 	client, err := redis.NewClientFrom(address)
 	if err != nil {
 		return nil, err
 	}
-	return &RevocationDB{
-		DB: client,
+	return &DB{
+		KVStore: client,
 	}, nil
 }
 
 // Get attempts to retrieve the most recent revocation for the given cert chain
 // (the  key used in the underlying database is the nodeID of the certificate chain).
-func (r RevocationDB) Get(ctx context.Context, chain []*x509.Certificate) (_ *extensions.Revocation, err error) {
+func (r DB) Get(ctx context.Context, chain []*x509.Certificate) (_ *extensions.Revocation, err error) {
```

Comment:
also nit - maybe change the receiver name from `r` to `db`. not a dealbreaker though.

Reply:
Good call: fixed.